scripts/tests/testIntegration.sh

```diff
@@ -4,4 +4,4 @@
 export NODE_ENV='test'
 
 echo 'Start firestore emulator and run integration tests:'
-firebase emulators:exec 'nyc mocha test/integration/**'
+firebase emulators:exec 'nyc --n=controllers/** mocha test/integration/**'
```

Comment:
The tests internally also cover all other folders. Is there any flag to ignore few folders, rather than include?

Reply:
There is --x or --exclude to ignore but there is a lot of folders to ignore.

Reply:
Updated here: https://github.com/Real-Dev-Squad/website-backend/issues/501#issuecomment-1048772568